### DISCUSSION TITLE
Deprecation of the master branch.

### DISCUSSION THREAD

**Discussion by tupoy-ya** [tupoy-ya/TupoyeMenu#58] [Announcements]
Soon i will change the default branch of this repository from master to lite.

### What does that mean?

The default branch of this repository will be changed to the lite branch and master will only receive basic merge commits, and critical fixes.
This also means a shift of aim on this project, instead of adding random untested shit into default branch i will try to keep the default branch clean and working.

If you have a fork of TupoyeMenu: You should try to merge lite, as master will lack behind and will not be updated.

If you want to use a feature that is removed from lite: I don't know.

### Why?
1. The main reason for this fork was to add untested broken features that crash your game, however in recent times YimMenu does this in their upstream repo so there is no reason to keep TupoyeMenu like this.
2. YimMenu became very bloated in recent times and now we have half working features that can be done in 1 line of Lua but instead are done directly in the menu. I don't think such functions are necessary and should be removed.
3. I think having a stable default branch is better then a half working one.

**Comment by ghost** [comment]
Finally, I'm glad someone sees how bloated and buggy yimmenu has become. Felt like I was going insane over there. Can't even use it anymore because it's so unstable. 

Btw, I tried out your fork earlier and I appreciate some of the changes you've done to the UI. I approve. Still needs work, but definitely a step in the right direction. I'm still testing to see if it suffers from some of the bugs I've found in yimmenu. Will report back when I have further mileage with it. 

edit: unfortunately this menu suffers from some of the same bugs yimmenu does. first one to pop out was how it prevents some npc waves from spawning during missions. haven't tested it enough to see if hitting random breakable objects while driving causes a ctd but i'll test it every now and then.
